.github/parm/use_case_groups.json

```diff
@@ -54,6 +54,11 @@
     "index_list": "5",
     "run": false
   },
+  {
+    "category": "clouds",
+    "index_list": "6",
```

Comment:
Let's update this to 8 to avoid the other two use cases (6 and 7) that have already gone in ahead of this one.

Reply:
Thanks @CPKalb !